docs/docs/how_to/using-devcontainers.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Devcontainers and Codespaces
+description: "Learn how to set up a devcontainer in your GitHub repository for a seamless coding experience with Codespaces. Follow our easy 8-step guide to create your own Noir environment without installing Nargo locally."
+keywords: ["Devcontainer", "Codespaces", "GitHub", "Noir Environment", "Docker Image", "Development Environment", "Remote Coding", "GitHub Codespaces", "Noir Programming", "Nargo", "VSCode Extensions", "Noirup"]
+sidebar_position: 1
+---
+
+Noir has extensive tooling. Thanks to Noirup, adding a devcontainer to your own repository is quite easy and pretty powerful.
+
```

Comment:
```suggestion

<video width="100%" height="auto" controls>
  <source src={require('@site/static/video/how-tos/devcontainer.mp4').default} type="video/mp4" />
  Your browser does not support the video tag.
</video>

```

Great to first demonstrate value, then explain how-to.

Reply:
Yep I can add it 👍

Reply:
I'll add it in [getting started](https://noir-lang.org/docs/dev/getting_started/tooling/#codespaces) as it seems more suitable for showcasing, leaving the how-to guide for instructions only